### ISSUE TITLE
Going fullscreen breaks HDR entirely

### ISSUE THREAD

**Issue by avidgameboy** [space-wizards/space-station-14#28822]
Description:
Entering fullscreen mode disables hdr in windows and if it is reenabled it simply doesnt work

Reproduction:
Enable HDR in windows, launch game, enter fullscreen mode

Screenshots:
N/A as hdr does not show in screenshots

Additional context:
I am on Windows 11

**Comment by FairlySadPanda** [comment]
Yep, this is a sucky known issue with Clyde (the Robust rendering engine) due to its window management.

**Comment by avidgameboy** [comment]
I did check but I couldn't see any other reports, sorry for timewasting with a known issue

**Comment by ShadowCommander** [comment]
Don't worry about whether or not it was a waste of time. If it wasn't easily findable, then it probably needed writing down or given more description so that it can be tracked easier.

**Comment by PJB3005** [comment]
Put it on the pile of reasons OpenGL is a broken turd and shouldn't be used in $current_year.

**Comment by avidgameboy** [comment]
> Don't worry about whether or not it was a waste of time. If it wasn't easily findable, then it probably needed writing down or given more description so that it can be tracked easier.

Thanks, you guys keep up the great work
